README.md

```diff
@@ -22,43 +21,81 @@ package](https://xclim.readthedocs.io/en/stable/) (Logan et al., 2021).  Built u
 dask, xclim benefits from simple to use parallelization and distributed computing tools 
 and can be easily deployed on High Performance Computing (HPC) environments.
 
+This repository contains the code used to generate and analyze the ESPO-R datasets. In
+addition to xclim and other freely available python libraries, it uses [xscen](https://github.com/Ouranosinc/xscen),
+a tool also developed at Ouranos to build climate data processing workflows. This tool
+as the particularity of relying on data catalogs as handled by [intake-esm](https://intake-esm.readthedocs.io/en/latest/index.html)
+as well as on YAML configuration files with a simple but specific structure. The catalog
+files and all paths needed by the configuration are missing from this repository, since
+they are specific to the data architecture of the computed running the code. To reproduce
+ESPO-R, one will need:
+
+- `CORDEX.json` and `CORDEX.csv` :An intake-esm catalog, compatible with xscen, listing the daily CORDEX datasets to use as inputs.
+- `project.json` and `project.csv`: An intake-esm catalog, compatible with xscen, listing the datasets created by this code.
```

Comment:
Your code doesn't create that ? It is not very clear that you need it to reproduce the code, but it lists the datasets created by the code..

Reply:
J'imagine que c'est parce que tu en as besoin pour l'analyse, mais le post-processing ne l'a pas créé parce que ça pré-date xscen? Ça mériterait une petite clarification.

Reply:
En effet c'est parce que ça pré-date xscen. En fait, je peux juste le créer dans le script d'ajustement.

Reply:
J'ai finalement ajouté une phrase, trop chicken de modifier le script _après_ que les données soient générées.